publish/check-for-published-rpm.sh

```diff
@@ -30,8 +30,18 @@ lsrpm | comm -23 - old-rpms.txt > new-rpms.txt
 
 # Check if there are any new RPMs.
 if [ -z "$(cat new-rpms.txt)" ]; then
-  echo "FAILED: aliPublish finished, but produced no new RPM for $PACKAGE_NAME" >&2
-  exit 1
+  cat >&2 << EOF
+
+WARNING: aliPublish finished, but produced no new RPM for $PACKAGE_NAME
+
+This may be because aliPublish failed, or because an existing package was
```

Comment:
I would mention also the fact you might have something wrong in aliPublish-rpms.conf.

Reply:
OK, updated!